src/proposals/20250821/SparkEthereum_20250821.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { IERC20 } from 'forge-std/interfaces/IERC20.sol';
+
+import { IMetaMorpho } from 'metamorpho/interfaces/IMetaMorpho.sol';
+
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+import { MainnetController }               from "spark-alm-controller/src/MainnetController.sol";
+import { RateLimitHelpers, RateLimitData } from "spark-alm-controller/src/RateLimitHelpers.sol";
+
+import { ICapAutomator } from "sparklend-cap-automator/interfaces/ICapAutomator.sol";
+
+import { SparkPayloadEthereum, IEngine, EngineFlags } from "../../SparkPayloadEthereum.sol";
+
+/**
+ * @title  August 21, 2025 Spark Ethereum Proposal
+ * @notice SparkLend:
+ *         - Refresh Market Params
+ *         Spark USDS Morpho Vault:
+ *         - Activate Vault Fee
+ *         Spark Treasury:
+ *         - Transfer Aave Revenue Share Payment
+ *         - Authorize Rate Limited Transfer of MORPHO Tokens to Multisig
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/august-21-2025-proposed-changes-to-spark-for-upcoming-spell/26997
+ * Vote:   https://vote.sky.money/polling/QmefEkAi
+ *         https://vote.sky.money/polling/QmP8NVR5
+ *         https://vote.sky.money/polling/QmNmGBSt
+ */
+contract SparkEthereum_20250821 is SparkPayloadEthereum {
+
+    uint256 internal constant MORPHO_SPARK_USDS_VAULT_FEE = 0.1e18;
+    uint256 internal constant USDS_AMOUNT                 = 19_411.17e18;
+
+    address internal constant AAVE_V3_COLLECTOR = 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c;
+    address internal constant MORPHO_TOKEN      = 0x58D97B57BB95320F9a05dC918Aef65434969c2B2;
+    address internal constant SPARK_MULTISIG    = 0x2E1b01adABB8D4981863394bEa23a1263CBaeDfC;
+
+    address internal constant NEW_USDS_DAI_IRM  = 0x8a95998639A34462A1FdAaaA5506F66F90Ef2fDd;
+    address internal constant NEW_USDC_USDT_IRM = 0x2961d766D71F33F6C5e6Ca8bA7d0Ca08E6452C92;
+    address internal constant NEW_WETH_IRM      = 0x4FD869adB651917D5c2591DD7128Ae6e1C24bDD5;
+
+    function collateralsUpdates()
+        public pure override returns (IEngine.CollateralUpdate[] memory)
+    {
+        IEngine.CollateralUpdate[] memory collateralUpdates = new IEngine.CollateralUpdate[](8);
+
+        collateralUpdates[0] = IEngine.CollateralUpdate({
```

Comment:
Don't have to change now but for next time should order in same order as forum post for easier review